docs/source/cuda/cuda_array_interface.rst

```diff
@@ -513,6 +513,14 @@ The following Python libraries have adopted the CUDA Array Interface:
 - `ArrayViews <https://github.com/xnd-project/arrayviews>`_
 - `JAX <https://jax.readthedocs.io/en/latest/index.html>`_
 - `PyCUDA <https://documen.tician.de/pycuda/tutorial.html#interoperability-with-other-libraries-using-the-cuda-array-interface>`_
+- `DALI: the NVIDIA Data Loading Library (DALI) <https://github.com/NVIDIA/DALI>`_ :
```

Comment:
Looking at the rendered docs, do you think we can make it:
```suggestion
- `DALI: the NVIDIA Data Loading Library <https://github.com/NVIDIA/DALI>`_ :
```
to avoid the repetition of DALI in the link?